packages/react-core/src/core/providers/thirdweb-provider.tsx

```diff
@@ -115,44 +113,34 @@ export const ThirdwebProviderCore = <
 >(
   props: React.PropsWithChildren<ThirdwebProviderCoreProps<TChains>>,
 ) => {
-  const _supportedChains =
+  const supportedChains_ =
```

Comment:
oof can we name this differently rather than supportedChains_ vs _supportedChains ? super hard to follow what's going on